LayoutTests/imported/w3c/web-platform-tests/css/css-text/text-autospace/text-autospace-dynamic-001-expected.html

```diff
@@ -1,4 +1,10 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<div>国国AA国国AA国国</div>
+<style>
+.no-as { text-autospace: no-autospace; }
```

Comment:
Shouldn't the selector here just be `div`?

Reply:
It could be, I think in the current way it shows more explicitly what the expected behavior is, since the test is toggling the `.no-as` class dynamically

Reply:
Yes, but, don't you want this rule to also apply to the second DIV?

Reply:
I think the idea of the test is that on the beggining one line has auto space applied and the other not, and then we invert the behavior dynamically

Reply:
Yes, but the reference should not have autospace applied to either line, right? So wouldn't it be better if the selector matched both?